volley/connectors/rsmq.py

```diff
@@ -133,12 +133,20 @@ def delete_message(self, message_id: str) -> bool:
         Raises a TimeoutError after attempts have been exhausted.
```

Comment:
I assume `queue.deleteMessage` returns a flag indicating if the message was actually deleted or not, either way we are certain the message is gone.

```suggestion
        Returns:
            Flag indicating the message was removed from the queue. 
```

Reply:
Totally fair. Thank you